RecoVertex/Configuration/python/RecoVertex_phase2_timing_cff.py

```diff
@@ -1,20 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 from RecoVertex.Configuration.RecoVertex_cff import unsortedOfflinePrimaryVertices, trackWithVertexRefSelector, trackRefsForJets, sortedPrimaryVertices, offlinePrimaryVertices, offlinePrimaryVerticesWithBS,vertexrecoTask
 
+from RecoVertex.PrimaryVertexProducer.TkClusParameters_cff import DA2D_vectParameters
+
 unsortedOfflinePrimaryVertices4D = unsortedOfflinePrimaryVertices.clone(
-    TkClusParameters = dict(
-        algorithm = "DA2D_vect", 
-        TkDAClusParameters = dict(
-            Tmin = 4.0, 
-            Tpurge = 4.0, 
-            Tstop = 2.0
-        ),
-    ),
+    TkClusParameters = DA2D_vectParameters,
```

Comment:
In all these cases please
```suggestion
    TkClusParameters = DA2D_vectParameters.clone(),
```
Otherwise if anything would modify the contents of `unsortedOfflinePrimaryVertices4D. TkClusParameters`, those modifications would apply to all other modules that had `TkClusParameters = DA2D_vectParameters`, which would lead to behavior that would be very difficult to understand.